sbag/generators/javascript/templates/app/src/app/app.module.ts.jinja

```diff
@@ -2,6 +2,7 @@ import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { BrowserModule } from '@angular/platform-browser';
 import { HttpClientModule } from '@angular/common/http';
+import {NgxPaginationModule} from 'ngx-pagination';
```

Comment:
Backend pagination maybe is a better solution here. What others think? @nikolina-sarenac  @veca22

Reply:
You're right, I will change that